lib/anycable-rails.rb

```diff
@@ -1,3 +1,7 @@
 # frozen_string_literal: true
 
 require "anycable/rails"
+
+module Anycable
+  class CompatibilityError < StandardError; end
```

Comment:
The error class shouldn't be defined here, we don't want to load the whole `anycable-rails` when loading compatibility tools.

Let's move it to `anycable/rails/compatibility.rb`

Reply:
done